.github/workflows/macos.yml

```diff
@@ -26,9 +26,9 @@ jobs:
 
       - name: Install packages
         run: |
-          brew install automake help2man texinfo bison berkeley-db@4
+          brew install automake libtool help2man texinfo bison berkeley-db@4
           opt="/usr/local/opt"; \
-          for d in automake help2man bison texinfo berkeley-db@4; do \
+          for d in automake libtool help2man bison texinfo berkeley-db@4; do \
             test -d "$opt/$d/bin" && echo "$opt/$d/bin" >> $GITHUB_PATH; \
             test -d "$opt/$d/lib" && export LDFLAGS="-L$opt/$d/lib $LDFLAGS"; \
             test -d "$opt/$d/include" && export CPPFLAGS="-I$opt/$d/include $CPPFLAGS"; \
```

Comment:
in theory pkg-config should find and setup lib and include directories correctly - you may want to add pkgconfig from brew (install and PATH) and see all the beauty of running with the correct setup (in theory); from the list above only berkeley-db would go to lib and include, so you may want to separate that out to an additional loop

Reply:
Ah, indeed. I was just trying to (minimally) fix whatever was already there, but this is a useful improvement.